jwala-webapp/src/main/webapp/resources/js/alert.js

```diff
@@ -1,10 +1,10 @@
 /**
  * sources: http://coding.abel.nu/2012/01/jquery-ui-replacement-for-alert/
  */
-$.extend({ alert: function (message, aTitle, aModal) {
+$.extend({ alert: function (message, aTitle, aModal, closeCallback) {
   $("<div></div>").dialog( {
-    buttons: { "Ok": function () { $(this).dialog("close"); } },
-    close: function (event, ui) { $(this).remove(); },
+    buttons: { "Ok": function () { $(this).dialog("close"); closeCallback(); } },
```

Comment:
is the clockCallback a required parameter now? Do we need to check if it's undefined before calling it?